packages/sdk/dist/index.js

```diff
@@ -0,0 +1,77 @@
+var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {
+    function adopt(value) { return value instanceof P ? value : new P(function (resolve) { resolve(value); }); }
+    return new (P || (P = Promise))(function (resolve, reject) {
+        function fulfilled(value) { try { step(generator.next(value)); } catch (e) { reject(e); } }
+        function rejected(value) { try { step(generator["throw"](value)); } catch (e) { reject(e); } }
+        function step(result) { result.done ? resolve(result.value) : adopt(result.value).then(fulfilled, rejected); }
+        step((generator = generator.apply(thisArg, _arguments || [])).next());
+    });
+};
+export function createClient(opts) {
+    return new ServerClient(opts);
+}
+class ServerClient {
+    constructor(opts) {
+        this.environment = opts.environment;
+        this.secretKey = opts.secretKey;
+        this.publicKey = opts.publicKey;
+        const { apiHost = "pipedream.com" } = opts;
```

Comment:
Needs to be changed to api.pipedream.com

Reply:
done in new branch